pyrogram/client/dispatcher/dispatcher.py

```diff
@@ -161,6 +174,22 @@ def update_worker(self):
                                                  else None)
                         )
                     )
+
+                elif isinstance(update, Dispatcher.DELETE_MESSAGE_UPDATES):
```

Comment:
I was about to suggest to put this branch into the above, but then I realized that `parse_messages` would require some changes to also parse deleted messages. Using another utility function makes sense here. Good job!